plugins/jobs/web_client/JobStatus.js

```diff
@@ -47,12 +53,20 @@ var JobStatus = {
     registerStatus: function (status) {
         _.each(status, function (info, name) {
             this[name] = info.value;
-            this._map[info.value] = {
+
+            let statusInfo = {
                 text: info.text,
                 icon: info.icon,
                 color: info.color,
+                textColor: 'white',
                 finished: _.has(info, 'finished') ? info.finished : false
             };
+
+            if ('textColor' in info) {
```

Comment:
Thanks -- I think this is just something that was hammered into those of us who read Crockford's treatises on javascript: "only use `in` when you really mean it", to the point that when we do see it now, it raises a red flag, even if it probably doesn't matter.